src/minio.rs

```diff
@@ -312,6 +312,50 @@ impl Client {
                     .and_then(|s| xml::parse_bucket_list(s))
             })
     }
+
+    pub fn list_objects(
+        &self,
+        b: &str,
```

Comment:
maybe we should start calling the function parameters something more meaningful like `bucket_name` and such